Dfe.Data.SearchPrototype/Web/Tests/APITests/ApiSearchResults.cs

```diff
@@ -35,16 +36,16 @@ public async Task GetSwaggerPage()
     }
 
     [Theory]
-    [InlineData("Academy", 2)]
-    [InlineData("School", 1)]
+    [InlineData("Academy", 100)]
+    [InlineData("School", 100)]
```

Comment:
I'm not sure what this is testing any more - in both cases it seems to test that the results limit is hardcoded to 100 when there are more results than that

Reply:
The API is returning 100 results max. Is this expected?

Reply:
It's hardcoded to return a maximum of 100, because there's no pagination. Because you're searching for a very general term that would have >100 results, you get 100. My main observation is if you want to test it, ok, but do you need to test it twice - on 'academy' and 'school'

Reply:
Understood. I'll remove one of them.